modal/functions.py

```diff
@@ -633,6 +634,7 @@ def from_args(
         if proxy:
             # HACK: remove this once we stop using ssh tunnels for this.
             if image:
+                # TODO(elias): this will cause an error if users use prior `.add_local_*` commands without copy=True
```

Comment:
I think we can remove the autossh installation - checking with @pawalt and @luiscape now